cvat/apps/events/handlers.py

```diff
@@ -31,6 +30,8 @@
 
 from .cache import get_cache
 from .event import event_scope, record_server_event
+from .const import WORKING_TIME_RESOLUTION, WORKING_TIME_SCOPE
+from .utils import calc_working_time_per_ids
```

Comment:
```suggestion
from .utils import compute_working_time_per_ids
```
Please avoid using truncated words, if possible.